docs/naps/2-conda-based-packaging.md

```diff
@@ -407,6 +425,15 @@ The full list of PRs is available on
 [this issue comment](https://github.com/napari/napari/issues/4248#issuecomment-1066574137).
 A complete list of features can be found in the packaging documentation [^napari-packaging-docs].
 
+#### Tasks
+
+* [ ] Create `napari/packaging` and migrate the conda bundle CI there, along with the relevant
```

Comment:
Very excited for this! Once this PR goes in we will create the new repo and grant necessary permissions.